### PR TITLE
support head method as valid method

### PR DESCRIPTION
**Pull request by spencer-codaio**
adds support for head method for https://golinks.io/bug/22391

corresponding `coda` PR: https://github.com/coda/coda/pull/71329

@coda/packs cc @huayang-codaio @jonathan-codaio @ekoleda-codaio

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,9 +3,11 @@
 This changelog keeps track of all changes to the Packs SDK. We follow conventions from [keepachangelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+
 ### Changed
 
 - **Future Breaking Change** Test framework will no longer normalize object output using the schema. (This will continue to work when executing the pack within Coda)
+- Added support for making `HEAD` HTTP requests using the fetcher.
```
Comment (ekoleda-codaio) [inline CHANGELOG.md:10]: Nitpick: Should this be under an "Added" heading instead, since we're adding a new feature?

Comment (spencer-codaio) [reply CHANGELOG.md:10]: oops good catch

File: api_types.ts [hunk 2]
```diff
@@ -460,7 +460,7 @@ export interface Network {
 }
 
 /** The HTTP methods (verbs) supported by the fetcher. */
-export const ValidFetchMethods = ['GET', 'PATCH', 'POST', 'PUT', 'DELETE'] as const;
+export const ValidFetchMethods = ['GET', 'PATCH', 'POST', 'PUT', 'DELETE', 'HEAD'] as const;
```
Comment (huayang-codaio) [inline api_types.ts:463]: I think you also need some changes in the coda repo.

**Review by huayang-codaio — APPROVED** [review]
lgtm except what Eric pointed out.

File: api_types.ts (hunk 2 above)
Comment (spencer-codaio) [reply api_types.ts:463]: yeah going to link that pr before merging this

**Comment by huayang-codaio** [comment]
maybe update `helpers/external-api/v1.ts`

**Review by jonathan-codaio — APPROVED** [review]

**Pull request merged**